<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1303</width>
    <height>1541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="wao_selectScreen">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Phase - Atmos</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Phase - WFS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Spots - WFS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Centroids - WFS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Slopes - WFS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Phase - Target</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Phase - DM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PSF LE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PSF SE</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="wao_selectNumber">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="wao_loadConfig">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>LOAD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="wao_init">
       <property name="text">
        <string>Init</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="wao_run">
       <property name="text">
        <string>Run</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="wao_next">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="wao_unzoom">
       <property name="text">
        <string>Unzoom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="wao_atmosphere">
       <property name="text">
        <string>Atmosphere</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="wao_resetSR">
       <property name="text">
        <string>Reset strehl</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>1271</width>
      <height>1051</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_20">
     <item>
      <widget class="GraphicsLayoutWidget" name="wao_pgwindow">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>900</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QCheckBox" name="wao_Display">
         <property name="text">
          <string>Use Display +Fr:</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="wao_frameRate">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>25</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="wao_PSFlogscale">
         <property name="text">
          <string>PSF log scale</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="StatsInTerminal">
         <property name="text">
          <string>Stats in terminal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="wao_autoscale">
         <property name="text">
          <string>autoscale</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Loop Freq (Hz):</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="wao_currentFreq">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="maximum">
          <double>1000000000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="MatplotlibWidget" name="wao_rtcWindowMPL" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>1140</y>
      <width>363</width>
      <height>47</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>GPU Device</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="wao_deviceNumber"/>
     </item>
     <item>
      <widget class="QPushButton" name="wao_setActiveDevice">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>1220</y>
      <width>1273</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_58">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>SR Long exp.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="wao_strehlLE">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>SR Short exp.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="wao_strehlSE">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="wao_clearSR">
         <property name="text">
          <string>Clear SR</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="MatplotlibWidget" name="wao_SRPlotWindow" native="true">
       <property name="minimumSize">
        <size>
         <width>1000</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>400</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1303</width>
     <height>39</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>MatplotlibWidget</class>
   <extends>QWidget</extends>
   <header>matplotlibwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
